src/services/vehicleLoader.ts

```diff
@@ -0,0 +1,39 @@
+import { register } from "./actions";
+import { getDistanceFromProgress } from "./spacingEditor";
+import { forEachVehicle, VehicleSpan } from "./vehicleSpan";
+
+const execute = register<LoadVehicleArgs>("rve-load-car", loadVehicleSettings);
+
+export interface VehicleLoadSettings
+{
+    trackLocation:        CarTrackLocation;
+	trackProgress:        number;
+}
+
+interface LoadVehicleArgs
+{
+    targets:  VehicleSpan[];
+    settings: VehicleLoadSettings[];
+}
```

Comment:
Why these extra spaces in between each property key and type?

Reply:
too much golang in $CORPJOB sorry